dt_model/symbols/constraint.py

```diff
@@ -23,8 +23,8 @@ class Constraint:
 
     def __init__(
         self,
-        usage: Symbol,
-        capacity: Symbol,
+        usage: Index,
+        capacity: Index,
```

Comment:
This is consistent but it is also not so useful in terms of DX when using the model.